library_generation/test/integration_tests.py

```diff
@@ -308,6 +304,3 @@ def __recursive_diff_files(
                 sub_dcmp, diff_files, left_only, right_only, dirname + sub_dirname + "/"
             )
 
-    @classmethod
-    def __remove_docker_image(cls):
```

Comment:
Since the GitHub runner cleans itself up after running the ITs and we usually want to troubleshoot locally via this integration test, it may be more convenient to not have to build the image from scratch every time when debugging.